TODO.md

```diff
@@ -0,0 +1,7 @@
+# TODO
```

Comment:
I tend to create todo files as well, but these are items better tracked in GitHub issues rather than a file that's living in the repository.